pkgs/misc/lssecret/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, pkg-config
+, libsecret
+, fetchFromGitLab
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  name = "lssecret";
+  version = "722013dc";
+
+  src = fetchFromGitLab {
+    owner = "GrantMoyer";
+    repo = name;
+    rev = version;
+    hash = "sha256-KLtxRXySoL9hT/c+O2arVR68ZYz+6Ce72YYCz60yot4=";
+  };
+
+  installPhase = "install -Dsm555 lssecret $out/bin/lssecret";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libsecret ];
+
+  meta = {
+    description = "A tool to list passwords and other secrets stored using the org.freedesktop.secrets dbus api";
+    license = lib.licenses.unlicense;
```

Comment:
Needs platforms, maintainer, and homepage. See: https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers and https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages. If you cannot be a maintainer feel free to add me:
```
maintainers = with lib.maintainers; [jordanisaacs]
```